views/standard/spec/indicators.js

```diff
@@ -281,6 +281,18 @@ export default {
          node.append(tag);
       });
    },
+   age: function age(pull, node) {
+      const timeDifference = Date.now() - new Date(pull.created_at).getTime();
+      const daysSinceCreation = Math.ceil(timeDifference / (1000 * 3600 * 24));
+      const numDots = daysSinceCreation < 21 ? daysSinceCreation : 20;
```

Comment:
```suggestion
      const numDots = Math.min(daysSinceCreation, 20);
```

Alternative suggestion. A little more concise, and I think as readable.